check.py

```diff
@@ -170,8 +178,9 @@ def get_branch():
         sys.exit(0)
 
     elif force_pass == "false" and len(check_results[1]) > 0 :
-        print("Done. The following URLS did not pass:")
-        print("\x1b[31m" + "\n".join(check_results[1]) + "\x1b[0m")
+        print("\n\nDone. The following URLS did not pass:")
```

Comment:
I think the line jump here should be also present in the other cases where no list is printed just to highlight the end of the action checks.

Reply:
Sure thing, I will add this now.